README.md

```diff
@@ -12,8 +12,34 @@ warc2zim provides a way to convert WARC files to ZIM, storing the WARC payload a
 Additionally, the [ReplayWeb.page](https://replayweb.page) is also added to the ZIM, creating a self-contained ZIM
 that can render its content in a modern browser.
 
+## Capabilities
+
+While we would like to support as many websites as possible, making an offline archive of a website obviously has some limitations.
+
+Scenario which are known to work well:
+- HTML and CSS documents
+- JS manipulating the DOM and/or doing simple fetch (preferably GET) requests
+  - E.g. JS manipulating the DOM to modify images, fetch remote stuff (JSON data, ...) is supposed to work
+  - POST requests support is fairly limited (at best, scraper replays the same response as it has been recorded)
+- Puny-encoded hostnames
+- Encoded URL path
+- URL query string
+- URL fragments
+- JS modules
+- HTML base href
+
 ## Known limitations
 
+- Any web site expecting a server to store live data and wanting to modifying those data (form, read/write api, ...) is not supported
+- Website using dynamic resources (dynamic URLs) fetch based on user-agent configuration (e.g. viewport), timestamp, unique ID
+  - E.g. if the viewport size is sent in every requests to fetch website images, this will not work since the URL built during the scrape will most likely be different than the URL built when the end-user read the ZIM content, and the ZIM reader won't find associated resource
+  - Scraper tries to do its best on few popular websites (e.g. Youtube) with fuzzy rules, but support is fairly very limited
```

Comment:
This statement has limited value to end user. What does this fuzzy mean? Problem/solution